src/generator/AutoRest.Ruby.Azure/Model/CompositeTypeRba.cs

```diff
@@ -16,9 +16,9 @@ namespace AutoRest.Ruby.Azure.Model
     /// </summary>
     public class CompositeTypeRba : CompositeTypeRb
     {
-        public static readonly Regex nameRegEx         = new Regex(@"^(RESOURCE|SUBRESOURCE)$", RegexOptions.IgnoreCase);
-        private static readonly Regex subResourceRegEx = new Regex(@"^(ID)$", RegexOptions.IgnoreCase);
-        private static readonly Regex resourceRegEx    = new Regex(@"^(ID|NAME|TYPE|LOCATION|TAGS)$", RegexOptions.IgnoreCase);
+        public  static readonly Regex resourceOrSubResourceRegEx = new Regex(@"^(RESOURCE|SUBRESOURCE)$", RegexOptions.IgnoreCase);
```

Comment:
Another minor: why are you formatting all = with vertical alignment? Again not a standard I've seen so for. 

Example: https://github.com/Azure/autorest/blob/285ba60fd8df7c71ecc73eed18285f6c78e32812/src/core/AutoRest.Core/Model/Method.cs#L25

I don't want to be a blocker here but I like consistency so if you find too many minor comments please bear with me :)